### DISCUSSION TITLE
can you please tell me how can I make new database for jobrunr so that all the jobrunr data can be stored there and It can be shared among different instances when hosted?

### DISCUSSION THREAD

**Discussion by ByteFlux88** [jobrunr/jobrunr#873] [General]
can you please tell me how can I make new database for jobrunr so that all the jobrunr data can be stored there and It can be shared among different instances when hosted?
I don't want to use InMemoryStorageProvider

**Comment by rdehuyss** [comment]
Have you read the docs?
